content/rs/administering/active-active.md

```diff
@@ -95,54 +74,62 @@ Do you want to set up NTP time synchronization now [Y/N]? Y
-the [Discovery
-Service]({{< relref "/rs/concepts/data-access/discovery-service.md" >}})
-for connections. FQDNs must be set up for each cluster per the [regular
-Redis Enterprise Software cluster
-configuration]({{< relref "/rs/administering/_index.md" >}})
-before a CRDB can be configured.
-
-### Network Ports
-
-For initial configuration and ongoing maintenance of a CRDB, each node
-of every cluster must have access to the Rest API ports of each of the
-other cluster's nodes.
-
-For synchronization, CRDBs operate over the standard endpoint ports. For
-example, when you created the CRDB, if you configured its endpoint port
-number to be 12845, that is the endpoint of the proxy for that CRDB on
-each cluster and must be opened.
-
-See [Network Port
-Configuration]({{< relref "/rs/administering/designing-production/networking/port-configurations.md" >}})
-for specifics on what ports should be open for VPNs and Security groups.
-
-### **Data Persistence**
-
-After creating CRDB instances on each participating cluster, you can
-individually set your data persistence configuration. Just like regular
-Redis Enterprise databases, CRDB data persistence options include AOF
-(Append-Only File) data persistence and
-snapshot.
-
-### Syncer process
-
-Each node in a cluster containing a CRDB Instance hosts a process called
-syncer. Its job is to synchronize data from other Instances of the CRDB.
-It connects to the other cluster's proxy and reads data from that
-database, brings it back and then writes it to the appropriate master
-shard of that database.
+## Network Configurations
+
+RS assumes that networking between the clusters is already configured when you create a CRDB.
+For security purposes, recommend that you configure a secure VPN between all clusters that host a CRDB instance.
+The setup of the CRDB fails if there is no connectivity between the clusters.
+
+## Network Ports
+
+For initial configuration and ongoing maintenance of a CRDB, every nude must have access to the REST API ports of every other node.
+You must also open ports for [VPNs and Security groups]({{< relref "/rs/administering/designing-production/networking/port-configurations.md" >}}).
+
+For synchronization, CRDBs operate over the standard endpoint ports.
+The endpoint port that you configure when you create the CRDB is the endpoint port of the proxy for that CRDB on each cluster.
+
+### Data Persistence
+
+You can set the data persistence configuration, including AOF (Append-Only File) data persistence and snapshot,
+for each participating cluster.
+
+## Syncer process
+
+Each node in a cluster containing a CRDB instance hosts a process called syncer.
+The syncer process:
+
+1. Connects to the other cluster proxy
+1. Reads data from that database
+1. Writes the data to the master shard of that database
+
+This process is also used in the [open-source redis replication](https://redis.io/topics/replication).
```

Comment:
the syncer is an RL process, it does not exist in OSS

Reply:
How about:
Some replication capabilities are also included in [open source redis] (https://redis.io/topics/replication).